src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -111,7 +111,7 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 		return connection;
 	}
 
-	public $connect(connectionProfile: IConnectionProfile, saveConnection: boolean = false, showDashboard: boolean = false): Thenable<azdata.ConnectionResult> {
+	public $connect(connectionProfile: IConnectionProfile, saveConnection: boolean = true, showDashboard: boolean = true): Thenable<azdata.ConnectionResult> {
```

Comment:
Same here. I think saving connection by default is fine, however.

Reply:
See above.